eiffel-vocabulary/EiffelAnnouncementAcknowledgedEvent.md

```diff
@@ -19,7 +19,36 @@
 The EiffelServiceDeployedEvent is to declare that an EiffelAnnouncementPublishedEvent has been received & acknowledged by a person or a machine.
 
 ## Data Members
- 
+### data.acknowledgement
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The Acknowledgement response.
+
+### data.user
+__Type:__ String  
+__Required:__ No  
+__Description:__ The identity of the user or entity that acknowledged the EiffelAnnouncementPublishedEvent. 
```

Comment:
Should Description be renamed to something less "generic"? Some form of "identity"?